src/geobr.jl

```diff
@@ -0,0 +1,590 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+"""
+Provides functions to download data from the GeoBR database:
+
+`GeoBR.extractmetadata`
+`GeoBR.download`
+`GeoBR.get`
```

Comment:
these three functions are not intended for end-users, we can remove from the list